types/handsontable/index.d.ts

```diff
@@ -494,19 +494,16 @@ declare namespace ht {
 }
 
 interface Handsontable {
-    new (element: Element, options: ht.Options): ht.Methods;
-    plugins: ht.Plugins;
-    hooks: ht.Hooks;
-    Dom: ht.Dom;
-    dom: ht.Dom;
-    helper: ht.Helper;
-    utils: ht.Utils;
-    renderers: ht.Renderers;
-    editors: ht.Editors;
+    new (element: Element, options: Handsontable.Options): Handsontable.Methods;
+    plugins: Handsontable.Plugins;
+    hooks: Handsontable.Hooks;
+    Dom: Handsontable.Dom;
+    dom: Handsontable.Dom;
+    helper: Handsontable.Helper;
+    utils: Handsontable.Utils;
+    renderers: Handsontable.Renderers;
+    editors: Handsontable.Editors;
```

Comment:
It's a matter of style, but you need some value that can be newed, either a `class Handsontable` or a `var` with a type that has a `new` method.

I like the class, but it does require a lot of copying of methods to prove that it implements Handsontable.Methods. Which you choose is up to you.